app/controllers/users_controller.rb

```diff
@@ -53,4 +54,21 @@ def handle_invitation
         invitation.delete
       end
     end
+
+    def handle_credit_card_charge
+      Stripe.api_key = ENV["STRIPE_TEST_SECRET_KEY"]
+
+      token = params[:stripeToken]
+
+      begin
+        charge = Stripe::Charge.create(
+          :amount => 999, # amount in cents, again
+          :currency => "usd",
+          :source => token,
+          :description => "Sign up charge for #{@user.email}."
+        )
+      rescue Stripe::CardError => e
+        # The card has been declined
```

Comment:
You are not doing anything with the error here. If the card is declined for some reason the user will not see the error. The comment is useless.